app/views/application/_footer.html.erb

```diff
@@ -2,14 +2,13 @@
   <div class="container">
     <div class="row">
       <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4 footer-left-col">
-        <a href="http://community.coopdevs.org/" target="_blank"><%= t('layouts.application.help') %></a>
+        <a href="https://adbdt.org/contacta/" target="_blank"><%= t('layouts.application.help') %></a>
         <%= link_to t('application.terms_conditions'), tnc_path, data: { target: '#modal', toggle: 'modal' } %>
       </div>
       <div class="col-xs-12 col-sm-2 col-md-2 col-lg-4 footer-center-col">
         <%= render 'application/menus/language_switcher' %>
       </div>
       <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4 footer-right-col">
-        <a href="https://twitter.com/timeoverflow" target="_blank">Twitter</a>
```

Comment:
no more Twitter account?

Reply:
We don't use it, so I prefer not to have this link